en/components/grids_templates/keyboard_navigation.md

```diff
@@ -22,35 +22,77 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 
 ## @@igComponent Keyboard Navigation
 
-Keyboard navigation is available by default in any grid and aims at covering as many as possible features and scenarios for the end user. When you focus a specific cell and press one of the following key combinations, the described behavior is performed.
+Keyboard navigation is available by default in the @@igxName and aims at covering as many as possible features and scenarios for the end user. Now in order to improve the overall page navigation experience we reduced the tab stops in the grid. Currently we have only the following main tab stops:
+    - GroupBy / Toolbar area /if is enabled/;
+    - header of the grid;
+    - body of the grid;
+    - column summaries /if are enabled/;
+    - the paginator /if is enabled/;
 
-### Key combinations
-@@if (igxName === 'IgxHierarchicalGrid') {
- - `Arrow Up` - navigates one cell up, going up the grid hierarchy if necessary (no wrapping);
- - `Arrow Down` - navigates one cell down, going deeper into the grid hierarchy if necessary (no wrapping);}@@if (igxName === 'IgxGrid' || igxName === 'IgxTreeGrid') {
- - `Arrow Up` - navigates one cell up (no wrapping);
- - `Arrow Down` - navigates one cell down (no wrapping);}
+> Note:
+> Due to this behavior change to navigate between the cells with `tab` and `shift + tab` key is no longer supported in the @@igxName.
+> By pressing `tab` key you are going through the following sequence: "Grid Toolbar" -> "Grid Headers" -> "Grid Body" -> "Summaries" -> "Footer / Grid
+> Paginator". Also keep in mind that if you add a focusable element in the grid template it will be added to the page tab sequence.
+
+### Header Navigation
+
+We introduced a full keyboard navigation support in the grid headers, so now is possible to navigate with the arrows keys through them. Also we provide a number of key combinations that trigger a different column functionality like filtering, sorting, grouping and etc.
+When you focus the header container of the grid and press one of the following key combinations, the described behavior is performed.
+
```

Comment:
It has been introduced a full **keyboard navigation support** in the @@igxName header so that it can be easily traversed with the arrow keys. Additionally, there are some key combinations that could trigger a different column functionality like `filtering`, `sorting`, `grouping`, etc. Focusing the header of the  @@igxName activates the described combinations below.